neurom/view/view.py

```diff
@@ -722,8 +722,8 @@ def dendrogram(obj, show_diameters=True, new_fig=True, new_axes=True, subplot=Fa
     kwargs['xlim'] = [- dnd.dims[0][0] * 0.5, dnd.dims[-1][0] * 0.5 + displacement]
 
     kwargs['title'] = kwargs.get('title', 'Morphology Dendrogram')
-    kwargs['xlabel'] = kwargs.get('xlabel', '')
-    kwargs['ylabel'] = kwargs.get('ylabel', '')
+    kwargs['xlabel'] = kwargs.get('xlabel', 'micrometers (um)')
+    kwargs['ylabel'] = kwargs.get('ylabel', 'micrometers (um)')
```

Comment:
Can we just add "unit of length" instead? Then it is up to the caller to set these. At least that way they know if is a length and not something else.